src/components/Terminal.vue

```diff
@@ -159,7 +159,7 @@ export default {
       const { cli } = this.$refs;
       if (!cli) return;
       const input = cli.input;
-      terminal.addToHistory(this.pwd, status, input, output);
+      if(output != "clear")terminal.addToHistory(this.pwd, status, input, output);
```

Comment:
Instead of this way of string matching, it's better to create a new class, like **NullOutput**, which will return null in **.html()** function call. Then you could just check if `output !== null` here.